test/common.py

```diff
@@ -1400,7 +1400,7 @@ def build(self, filename, libraries=None, includes=None, force_c=False, emcc_arg
       output = output_basename + output_suffix
     else:
       output = shared.unsuffixed_basename(filename) + output_suffix
-    cmd = compiler + [filename, '-o', output] + self.get_emcc_args(main_file=True)
+    cmd = compiler + [str(filename), '-o', output] + self.get_emcc_args(main_file=True)
```

Comment:
Why is this change needed?

Reply:
The official shlex.join only accepts strings and not `Path` objects.  Our version had the call to `os.fspath` in it.